x/dex/types/orders.go

```diff
@@ -3,49 +3,145 @@ package types
 import (
```

Comment:
can we add unit tests for new functions in this file

Reply:
added